content/en/logs/workspaces/use_cases/correlate_access_error_logs.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Correlate Web Access Logs and Web Error Logs
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Use case
+
+Log Workspaces enable you to integrate and correlate Apache Web Access Logs and Web Error Logs, providing clear insights into how specific errors impact users. Analyse these logs togehter to identify which user actions cause errors and address issues proactively. Follow these steps to monitor and correlate Apache Web access and Error logs using the flexible querying and visualization options available within Workspaces.
+
+## Setup
+
+This guide assumes that you are:
+- Submitting logs to Datadog for a similar use case.
```

Comment:
"Already" submitting logs maybe? I think I'd like to get a clearer idea of the current state here.

Reply:
```suggestion
- Already submitting logs to Datadog for a similar use case.
```